docs/pages/includes/config-reference/instance-wide.yaml

```diff
@@ -55,10 +55,18 @@ teleport:
         # token_name: /var/lib/teleport/tokenjoin
         token_name: "token-name"
 
-    # Optional CA pin of the auth server. This enables a more secure way of
-    # adding new nodes to a cluster. See "Adding Nodes to your Cluster"
-    # (https://goteleport.com/docs/setup/admin/adding-nodes).
-    ca_pin: "sha256:7e12c17c20d9cb504bbcb3f0236be3f446861f1396dcbb44425fe28ec1c108f1"
+    # Optional CA pin of the Auth Service, which enables new agents to trust a
+    # Teleport cluster when joining via the Auth Service directly. This can be
```

Comment:
```suggestion
    # Teleport cluster when joining from the Auth Service directly. You can specify the CA pin
```

Reply:
- I think "via" is more appropriate here since the joining would originate from the agent, rather than from the Auth Service, but take place through the Auth Service.
- I've made another suggestion for how to avoid saying "This" in the second sentence without saying "specify" as much